### DISCUSSION TITLE
Ploting on different machines not connected to full node

### DISCUSSION THREAD

**Discussion by happycouak** [Chia-Network/chia-blockchain#3691] [Q&A] [answered]
Hello,

I have a full node at home which only runs chia daemon and storage for plots, and one remote machine that only produces plots and send them over the internet to the full node.

The remote machine don't run any chia daemon, the only things done are:

- installing chia
- run a chia init (needed by chia plots create)
- do standards plots with --farmer_public_key and --pool_public_key from full node
- send them to full node

Does plots produced on remote node would actually be usable on full node to farm chia ?
When plots a received on the full node, they are well discovered so it seems fine, but I worry a bit about the fact that if a plot match there will be eligible for a chia ?

**Comment by nomadengineer** [comment, marked as answer]
Your setup seems ok, if the same nemonic and ca certs are used to initialize the chia client on plotting machine.
you check the plots on your main machine through _chia plots check_
